Prox/Prox/Widgets/LocationLoadingView.swift

```diff
@@ -93,19 +94,51 @@ class LocationLoadingView: UIView {
             self.dot.centerYAnchor.constraint(equalTo: self.centerYAnchor)
         ]
         NSLayoutConstraint.activate(dotConstraints, translatesAutoresizingMaskIntoConstraints: false)
+        startPulsing()
+
+        NotificationCenter.default.addObserver(self,
+                                               selector: #selector(LocationLoadingView.didEnterBackground),
+                                               name: .UIApplicationDidEnterBackground,
+                                               object: nil)
+        NotificationCenter.default.addObserver(self,
+                                               selector: #selector(LocationLoadingView.willEnterForeground),
+                                               name: .UIApplicationWillEnterForeground,
+                                               object: nil)
+    }
+
+    deinit {
+        NotificationCenter.default.removeObserver(self, name: .UIApplicationDidEnterBackground, object: nil)
+        NotificationCenter.default.removeObserver(self, name: .UIApplicationWillEnterForeground, object: nil)
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override var intrinsicContentSize: CGSize {
+        return CGSize(width: 100, height: 100)
+    }
 
+    fileprivate func startPulsing() {
         let aAnim = animation(fromState: .shrunk, toState: .enlarged)
         self.circleA.add(aAnim, forKey: "a")
 
         let bAnim = animation(fromState: .enlarged, toState: .shrunk)
         self.circleB.add(bAnim, forKey: "b")
     }
 
-    required init?(coder aDecoder: NSCoder) {
-        fatalError("init(coder:) has not been implemented")
+    fileprivate func stopPulsing() {
```

Comment:
Not needed but I figured if we have a start we should probably have an explicit stop.